xtra-libp2p-offer/src/current/maker.rs

```diff
@@ -75,9 +75,11 @@ impl Actor {
         let quiet = quiet_spans::sometimes_quiet_children();
         for peer_id in self.connected_peers.iter().copied() {
             self.send_offers(peer_id, msg.0.clone(), ctx)
-                .instrument(quiet.in_scope(|| {
```

Comment:
🤔 I just starting to read through the tracing library and this might be a silly question, but why do we instrument here explicitly instead of simple annotating the send offers method?

Reply:
I'm not sure either! @Restioson, what do you think?

Reply:
The main reason is because we want the span to be a child of `quiet` so that it is silenced. That being said, we might be able to achieve this through instrumenting send_offers and then using `.instrument(quiet)`

Reply:
See #2851